src/services/expenses.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+let expenses = [];
+
+function getAllExpenses() {
+  return expenses;
+}
+
+function findExpensesById(expensesId) {
+  return expenses.find(({ id }) => id === expensesId);
+}
+
+function createExpense(dataExpense) {
+  const newExpenseId = expenses.length + 1;
+
+  const newExpense = {
+    id: newExpenseId,
+    ...dataExpense,
+  };
+
+  expenses.push(newExpense);
+
+  return newExpense;
+}
+
+function removeExpense(expensesId) {
+  expenses = expenses.filter(({ id }) => id !== +expensesId);
```

Comment:
Rename 'expensesId' to 'expenseId' to better reflect its purpose. Also, use 'Number()' instead of '+' to convert 'expenseId' to a number for better readability.